test/integration/sf.e2e.ts

```diff
@@ -1,6 +1,15 @@
 import * as os from 'os'
 import {expect} from 'chai'
 import {Executor, setup} from './util'
+import StripAnsi = require('strip-ansi')
+const stripAnsi: typeof StripAnsi = require('strip-ansi')
+
+const chalk = require('chalk')
+chalk.level = 0
+
+function parseJson(json: string) {
```

Comment:
Running tests locally I was getting colored JSON results, so stripping those chars from the response.